### DISCUSSION TITLE
Second Renaissance 101 Course: Get a handle on the metacrisis and how we can transcend it in 7 days

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#922] [Ideas]
- What: A 7-10 episode series introducing the second renaissance: paradigm shifts, our civilizational unwinding and how we can transcend it
- For Who: People who want a more accurate/useful understanding of the state we're in and what we can do about it 😉
- Intention: Give people a framework to make sense of what is happening - and therefore hope, grounding and a context for better action

![image](https://github.com/life-itself/community/assets/180658/80505700-bbc5-47a2-9fd1-79d1d044d9b0)

## Asides

- How modernity got built - because understanding the last paradigm shift helps you see and understand how a new one can come to be
- Blindspots: what is in the way of transformation
  - Blindspots come from trauma
    - Greatest trauma is religion
       - Greatest blindspot is secularism
          - => “healing religion” (another course?) 

### Taglines

- Get a handle on the metacrisis in 7 days
- Discover what’s really driving the climate crisis and what we can do about it
- Understand the predicament we are in and how we can transcend it
- We are in a time between worlds. Get a handle on what is happening in this 7 day course.

## Episode sketch

1. [Lead-in/trailer] **The great unwinding/crisis/unravelling aka "Cracks in the foundations"**:. Issues we have are foundational. Metaphor: innocuous crack in the wall on the 20th floor actually indicates a fundamental problem in foundations => you need to move to a new building.
2. **"Fish in water": aka Intro to Paradigms**. What are they? The water we swim in. Bigger ans: culture i.e. onto-social paradigm. Views and values plus socio-economic system. 
3. **The first renaissance and the last great paradigm shift (pre-modern to modern)**
4. **Where are we? the great cycle of paradigms (or any complex system)**
    - birth, growth, maturity, decacy/death.
    - Look at visible signs e.g. climate crisis, lack of political vision
    - So where are we today?
5. So what can we do about it?
    - How are new paradigms born?
    - Religion the great taboo … and the essential ingredient in cultural evolution and social scaling in the past
6. Don’t make modernity wrong (what you resist, persists)
    - … And we need to mourn it … 
7. **Darkness before dawn: last time round we had the black death … this time well you know it already (climate crisis, out of control tech, pandemics etc).**
8. **The great awakening and the second renaissance**: a new dawn for a new civilization. Going from modernity to whatever comes next.
9. [Optional] How to get involved

**Comment by martinprogrammer** [comment]
@rufuspollock  [Lead-in/trailer] The great unwinding/crisis/unravelling aka "Cracks in the foundations":. Issues we have are foundational. Metaphor: innocuous crack in the wall on the 20th floor actually indicates a fundamental problem in foundations => you need to move to a new building.

I wonder how will the "Cracks in the foundations" be defined and depicted?  What do you think are the most worrying trends and the starkest examples?

**Reply by rufuspollock** [reply]
Great question - thank-you for asking for the clarification.

Here are some "cracks in the wall":

### Climate crisis (and ecological crisis more broadly)

What: obvious i think! Dangerous climate change and ecological degradation are accelerating. We continue to pump c02 into the atmosphere at a growing rate despite half a century of knowledge of the issue. Major vested interests actively oppose intervention and obscure the facts etc.

Superficial analysis: we just need more green energy ... let's get more solar.

Foundational issue: focus on endless material growth is core to capitalism and all major modern societies. Behind that materialism. tech obsession. Left brain delusions of control and simplified thinking (e.g. difficulty to grasp ecological dynamics and complexity etc).

### "Wellbeing gap (or decline)

Despite great material wealth (in "west") people not that happy and in many sectors wellbeing seems to be declining e.g. greater levels of suicide, depression.

Superficial analysis: a) dismiss issue (people actually are getting happier b) we just need more stuff c) (slightly deeper) we need to spend more on psychotherapy etc.

Foundational issue: (hyper versions of ...) materialism, individualism, secularism. Materialism: a dead end in terms of long-term growth in wellbeing. Hyper Individualism = atomization and loneliness. Secuarlism = no spirituality and no access to waking up which is best opportunity for profound wellbeing (see https://lifeitself.org/learn/fundamental-wellbeing)

### Polarization especially politically

Superficial "it's just social media" etc. 

Fundamentally: clash of world-historical cultural values systems as part of major paradigmatic transition from modernity to what comes after it (or in some cases still just from pre-modernity to modernity)

Concretely: (using Integral/spiral color metaphors) amber is clashing with orange/green. Orange is clashing with green. And teal is only just getting on the scene. See this book review for a more detailed analysis inspired by the Trump election https://lifeitself.org/blog/wilber-trump-post-truth-world-review-notes

Taking analogy of last great transition of medieval to modern: you had the great clash of catholicism with protestantism (major wars e.g. french wars of religion, 30 years war, spanish war in the low countries; massacres during all of these etc etc). You had the english civil war, the french revolution, the american revolution. These weren't easy transitions - there were bitter disagreements and extreme polarization.

### Growing inequality

What: inequality seems to growing systematically across a broad range of countries over the last half century.

Superficially just need a bit more redistribution.

Fundamentally related to the nature of the information economy and digital capitalism (or even capitalism in general!) - see https://openrevolution.net/book/ and e.g. https://lifeitself.org/blog/2018/06/08/can-digital-businesses-thrive-and-be-mindful

### Runaway tech

What: current AI development. clear risks but little ability to do much about it - in fact race to bottom (race to accelerate) seem to be getting worse.

Superficial analysis: need a bit more regulation.

Foundational analysis: we are tech obsessed (left-brain dominated, materialist), lack patience and ability to coordinate collective action problems (individualism etc).

### Wellbeing / mental health crisis

What: ongoing rise in suicide and depression in the materially *wealthier* countries.

...

### Meaning crisis

This a more ontological one - and hence more debatable.

What: loss of purpose, anomie, growth of nihilism.

Superficial analysis: always been this way ... or inevitable result of seeing our true nature (selfish genes, purposeless atoms) and (lack of) true purpose.

Foundation analysis: cancer of modernity that removed the magic from the world (mistakenly and erroneously).

**Reply by martinprogrammer** [reply]
@rufuspollock  

I'm not sure whether it goes with this particular narrative, but one of the "cracks" could be the crisis of democracy in the West and the rise of China, aided by the long-termism of their single party political system.  This slow but relatively steady ongoing historical event might also be signalling the succession of empires.  I wonder whether such high level, far reaching, global trends should be taken into consideration when homing in on the message?  The success of China fuels alternative ideologies which resonate with the "practical" far right and might lead to autocracy and authoritariansm.

We could analyse further what are the actual issues with our democracy, but that probably belongs to another discussion  ;-)

**Reply by rufuspollock** [reply]
@martinprogrammer excellent suggestion - i would agree that that would be another good example. It's demonstrated by brexit, trump etc which provide visible signs around this and broader malaise.

**Comment by matthewmccarthy11** [comment]
Question as it relates to number 3 and 4, or the 'two different kinds of renaissance'. To what degree are the two 'global'? While the first one occurred primarily in Europe, one can definitely say it started the 'modern world system'. I think sometimes the first renaissance is even referred to as a 'global renaissance' since it started the MWS. And there is a bit of a contradiction here where the 'world system' isn't evenly distributed of course and was originated on one part of one continent. With the second renaissance, it almost necessarily seems like it has to be a 'global' one, as it is concerned with things like planetary boundaries, planetary and global issues, and global cooperation. While the first renaissance was the 'start' of the 'modern world system' the second renaissance is something else, like the 'transformation' or 'maturity' of the 'world system', on a more global scale. There also is too the challenge of incorporating the global south in the 'second renaissance'. Whatever the case I think 'world systems analysis' generally is an interesting way to compare the two renaissances...'Awakening world system' is maybe an interesting way to consider or call the second renaissance 'world system.'

**Reply by rufuspollock** [reply]
@matthewmccarthy11 can you rewrite to split up paragraphs a bit and clarify the exact point you want to convey 😉

**Comment by martinprogrammer** [comment]
Again, One of the more visible concrete "crisis" is the mental health crisis.  It only highlights the underlying cracks, but probably strikes a chord with people and shapes up a need for transcending it.

1) The pace of changes, societal expectations and changing nature of work lead to stress, anxiety, and a feeling of uncertainty. 
2) Traditional values and belief systems that anchor us to reality and our identity and have provided stability are being questioned and, in some cases, challenged. 
The questioning of long-held values creates sense of disorientation and internal conflict, the loss of alignment with familiar principles.
3) The support for helping people weather the personal mental health crisis is not adequate

**Reply by rufuspollock** [reply]
Another excellent point - and usually i include that in the 3 "horsemen of the meta-apocalypse" - see e.g. https://lifeitself.org/blog/2020/06/22/bridging-the-wisdom-gap

I've tweaked my list above to include it.

As you mention i think it is a convergence of sociological stressors (greater loneliness etc) with more ontological stressors (anomie/lack of purpose etc).

**Comment by martinprogrammer** [comment]
@rufuspollock "For Who: People who want to true understanding of the state we're in and what we can do about it 😉"

1) Do we know who are these people?

2) Where to find them?

3) In order to find out more about our target audience - should we do a market research in order to:
    - create segmentation criteria
    - identify target segments
    - create an ideal representative of that segment
    - create a bespoke message
    - test the message and run it through iterations

4) If you agree with 3) above - more than one version will be needed

**Reply by rufuspollock** [reply]
@martinprogrammer great questions 👍

Simple answer is: "no, we don't exactly .. and that's why we are creating "beacons" like this as a way to locate folks ..."

And i agree we will need to iterate.

One helpful thing is brainstorming the places e.g. podcasts, fora, blogs etc where these kind of folks hang out ...

**Reply by rufuspollock** [reply]
@martinprogrammer what would be your answers to these questions - it's always interesting to hear ...

**Reply by martinprogrammer** [reply]
@rufuspollock  I'm jumping on your band wagon here - this is far beyond the scope of my enterprise and I'm very late to the table :-). I haven't got much ammunition to form answers to those questions, but I can ask some more.

1)  How to frame positive action out of raising metacrisis awareness?

2) Is there a particular (change) lever we're trying to pull with this course?  What action?

3) Who are the people who are most likely to have the power to move that lever?

4) What are their values and sensitivities?

5) Aligning the eco-system behind the message (so we don't have a myriad of these educational courses - and all of them offering alternative state of affairs ledger.

Dangers associated with getting it wrong

1) Polarisation of opinons, but also a risk of messages being seen as subversive and therefore -

2) Anticipating resistance to the message and the changes and

3) Mitigating risks of attempts to derail the change process (educational process)

**Reply by asimong** [reply]
Really, "For people who want to have a true understanding of the state we're in"??? Let's be frank here. There may be many people who fit that bill, but I wouldn't be chasing them. Personally (yes, I speak only for myself) I tend to be averse to people who want to give me a "true" understanding of anything. A useful understanding, yes. Something that I can work with to collaborate in changing the world for the better, yes. And yes, that does need narrative. But as soon as you put "truth" into the mix, you're inviting controversy.

I'm happy with people saying "this is what I live by and this is how it works for me". I can respect that. But (healing narratives again) different narratives work for different people, and there are good reasons for that, both in personal trauma and historical trauma. The more you say about your personal background and why you have adopted particular views and values, the easier it is for others to identify with you and go along with you; or (just as usefully) for people to say, that's not me I'll look elsewhere. You don't want those people in your course, they'll just be a disruptive influence.

**Reply by rufuspollock** [reply]
OK, and i've tweaked the description. Personally, whenever i write (or read) "true" or "wise" as an adjective we can take it to be "truer" and "wiser". Truth and wisdom are a mountain with no top - there is an up but not a top.

**Comment by martinprogrammer** [comment]
@rufuspollock Good session today.  

I just wanted to share my initial thinking, trying to define a common theme that could define those cracks in a more abstract sense.

- All areas and domains that we listed can be identified as "systems".  
- All of the systems are human designed, built and run

**Everything that we design is:**

1) for a particular reason - to meet objectives in order
2) to address particular problems and challenges
3) to be of use by particular audience
4) to the best of our abilities

**All those systems (areas and domains) should have integrity.**

Integrity means that there should be no contradictions built-in and that the system should be designed and run to:

1) serve particular mission and values
2) be transparent
3) be accountable
4) be responsive to changes (in order to improve the design, to broaden the objectives)
5) ensure participants act in good faith towards the common good  

To ensure positive participation in the system
1) so people become stakeholders
2) so the system is sustainable
3) people don't get disillusioned

If the integrity falls apart - the system becomes corrupt through:
1) participants losing faith in the system
2) losing focus of the common purpose

**My premise is that the "cracks" are the manifestation of the corruption of the systems.**
Which brings about an uncomfortable idea (suggestion) that maybe it's the system that takes precedent over individuals?

What do you think?  Is this helpful?

**Reply by asimong** [reply]
I'm very inclined to take on this line that the "real" distinction we need to look out for is between self-consistent systems on the one hand, and corrupt / inconsistent / self-contradictory / self-defeating systems on the other. On the other other hand, though, even if capitalism isn't fully consistent, I would have thought that absolutism is pretty well consistent. Just unjust, and absolutely not what we want (unless we have the absolute power!) Sadly, I suspect that one can actually frame an absolutist dictatorship as transparent, accountable, etc. in its own terms. Responsive to changes, sure — just eliminate the people who are trying to derail it! And ensure that participants act in good faith towards the common good (as I the dictator decree it as the common good, and who is to contradict that?) — if you act in good faith towards that you will be rewarded; if not you will be taken to some unpleasant isolated place out of reach of the rest of society. No, I don't want that  ….

**Reply by martinprogrammer** [reply]
@asimong 
Every project that we propose is defined by a motivation, agreement, specification and participation. 

Any changes to the project must be subject to an agreement and not arise from the corruption of the idea, process, execution...

Is that the absolutism that you're talking about?  Do you disagree with the above?

**Reply by martinprogrammer** [reply]
@asimong  We could have a very long conversation about the just and unjust:

1) Is positive discrimination just or unjust?
2) is being tolerant to close-mindedness just or unjust?
3) Is "a transparent and accountable absolutist dictatorship" more just than an absolutist dictatorship that pretends not to be one?

This is why I think open discussions are important - to establish a baseline and try and harmonise views.

**Reply by rufuspollock** [reply]
The word "system" is doing a lot of work here. Can you give examples of what you mean by systems e.g. is a culture a system? Is a human body a "system"?

I also think the word "design" is probably a bit of a stretch for a culture. At least for me, design implies conscious intentional choice. Most cultures aren't created that way - they arise out of a multitude of choices mostly unconscious and unintentional.

Now we can get *more* conscious about them, or at least about recognizing and adopting cultural patterns that "work" ... and that's partly what this course is about. However, that is recent and I'd still see that more like a gardener "shaping" and "curating" their garden than a designer "architecting".

**Reply by martinprogrammer** [reply]
@rufuspollock 

I was referring to the "Areas or domains" identified in the shared Google document. Listed below. 

They are all, in my opinion, systems - governance systems, economic systems, social systems, justice systems, environmental systems, health systems...

Education 
Economy
Health
Social Justice
Security
Ecosystems
Technology
Agriculture
Transportation 
‘Politics’
Mental Health / Trauma
Art & entertainment
Personal Growth & Development
Social Life/Socialising
Democracy & Governance
Housing
Architecture & urbanisation 
‘Social imaginary’

I'm happy to go down the culture route - and culture is not a system in isolation.  
It is an emergent property of social systems - it's the point where individual practices and beliefs, operating within a particular context,   converge into commonality.  It's a concept, rather than a system on this level.

But depending on its **intended role** in a higher level system - 

**it can become a system** because it will be weighed down by certain objectives and intentionality that will influence its further development.

What you're suggesting by "gardener shaping" indicates to me that you're treating culture as a system.

Regarding "design" - there's an inherent design in everything regardless of our inability to identify the designer or their motivation.
If you're building to a design - then what you built can be judged by how faithful it is to the design.  But if you're extracting design from the structure itself - you're just reading its specs (or discovering its design).

Anyway, this is just really abstract and **what I'm trying hard to do is being constructive and somehow contribute to the discourse.**  I still haven't found the right angle :-)

**Reply by rufuspollock** [reply]
I'm going to go back here now that your meaning of system is clear to a starting point of the thread which was:

> My premise is that the "cracks" are the manifestation of the corruption of the systems.

This goes back to the definition of systems which we spent the thread on. You *can* expand system to mean both "areas" of society (e.g. transportation) and entire "functional" (?) domains like "culture" or "technology". If so then this is kind of true but you have then enlarged system to mean pretty much everything.

The more specific claim we were making in the course is that (some of) the major cracks are foundational in the sense of relating to the core underlying cultural/ontological underpinnings of our current major paradigm (modernity). cf the thread above about cracks where we spelled out a bit why some things were foundational and how https://github.com/orgs/life-itself/discussions/922#discussioncomment-7681922

> Which brings about an uncomfortable idea (suggestion) that maybe it's the system that takes precedent over individuals?

It's definitely the case that I would argue that "systems" e.g. culture etc take some precendence over individuals in the overall "sweep of history". Though, of course, cultures / systems are made up of people so there is some kind of non-dual dance there too.

> Anyway, this is just really abstract and what I'm trying hard to do is being constructive and somehow contribute to the discourse. I still haven't found the right angle :-)

Just to say: your questions and reflections have been a big contribution so far e.g. we clarified "cracks" stuff a lot above.

**Reply by martinprogrammer** [reply]
@rufuspollock 
What are those core underpinnings that are affecting the cracks?
If you start skinning this onion in order to find the primary principles - you’ll go down to the atom level and lose any fixable context.

If you make changes at a deeper level - you’ll find that you’ll have a bigger job or re-writing the entire dependency ledger of the subsequent foundations of layered systems than if you fixed only the top layer and ignored what lies underneath.

Do you need to fix the underlying historical context in order to improve or replace the current system?

The corruption of the systems (systems that are manifesting dysfunctional behaviour) can be (one of the reasons) due to something we touched on recently - the distinction between compliance and engagement.  

1) Engagement with the systems would mean that we'd be fixing the systems as we come across problems because we're motivated to make the systems more faithful to their design manifesto (collective good) and we’d be acting in the spirit of the rules, rather than within the compliance criteria.

2) Compliance would mean that we would be actively testing the boundaries of the system's will to preserve its integrity by incorporate those faults into our game plan to spot opportunities and use them to our personal ends.

The system not fighting back is a corrupt system.
The participants in the system more interested in their own personal gain then in the preservation of the system also indicates that the system is corrupt.

How are we going to fix the systems?

1) Design the systems that are not self-undermining (no contradictions and awareness of what hurts it and what makes it stronger)
2) Make everyone motivated stakeholders in the system - so they don’t try to destroy it.

How are we going to do this?

By creating new, community based, alternative systems (just like you’re doing with intentional co-living) that will prove that they are better than what we already have.  

Then, we don’t have to change anything too much fundamentally because we already have the world with a built-in urge to adopt what’s better.

BTW - I loved your DAO and co-op podcasts and I think those concepts should play a major role.  I’m not talking about the technology itself but what they are prioritising -  transparency, reputation, integrity, accountability, common purpose. 

Also, I think that Balaji’s network state is a great perspective to take clues from.

**Comment by martinprogrammer** [comment]
@rufuspollock 

Further course feedback:

At the moment, I'm getting mixed messages with regards to the values and ideas that you're trying to convey.
Is the course supposed to be a consistent and unified vision or just an unordered bunch of things offered in order to "awaken" us from the slumber?

I found interesting Sylvie's recount of that "untitle campfire chat" about future.
If we imagine future as a (virtual) techno-utopia where we'll "transcend" our bodies and cultural and ethical boundaries of all sorts in order to be more in tune with each other and take communication and interconnectivity to another level - what's the negativity?  Is there a technophobic slant to to story?  Maybe I got it all wrong, but it's important to clearly outline what's the position, just in order to see if there's any hidden, unjustified bias?

Regarding the past - are we better off deriving our vision from the past or from the future?  Maybe it's better to work on the future design by using components of some lost jewels of the forgotten cultures?  

For me, at least, I would like to see an itemised list of what you suggest a "weller and wiser" world should look like and start from there.

Let me know if any of it helps.  If I were doing this course- I'd generally welcome interaction and feedback...

**Reply by asimong** [reply]
here's a high level idea for the course.

We've been talking about paradigm  / worldview / etc shifts in various areas: art, film, architecture, ... though we haven't tackled music yet. More about that separately. But what came to me is that, as different people have greater awareness of different cultural forms, I think it might work well to have one whole session as an extended breakout around separate forms.

**Reply by asimong** [reply]
Now for music. Are people familiar with the astounding talent of Jacob Collier? He seems to be wowing all kinds of people, including the teenage –young adult demographic (e.g. my daughters aged 17 and 23) but not only them. I would love a conversation around this version in particular of Little Blue https://youtu.be/IQvzX0Z3HE4?si=5L6ZzGwqCgqqQqWc with the question, how much meta-modern / yellow / teal (not blue :-) ) paradigm do people find in it?

I find music is a fascinating window on culture. Is a key a perspective, for example? Collier has not only the kind of atonality that characterises postmodern music, but integrates it with modern and pre-modern themes. Just get the lyrics, to start! And that performance itself, what does that embody? Material for much reflection.

**Reply by martinprogrammer** [reply]
@asimong  
I've just listened to this tune - i think it ranges very close to the lowest common denominator as far as the music goes.  It probably resonates with a lot of people and taps into some inner pain, hope, Weltschmerz... Please share what exactly do you find in this song and how do you think it fits into the metamodern paradigm?

**Reply by asimong** [reply]
other views, please?

**Reply by martinprogrammer** [reply]
How about this for eclecticism, tradition - world music 
https://www.youtube.com/watch?v=GTdVBv9xtsM

**Reply by asimong** [reply]
here's a related question, particularly for course attendees but also for anyone interested. Looking at the vast variety of popular, folk and contemporary music (including classical played in the present) which would you associate with which level / colour / paradigm?

**Reply by matthewmccarthy11** [reply]
I'm glad you're bringing music into the discussion @asimong, it's pretty surprising how we've never really touched on it before. However, this is the first time I'm listening to Jacob Collier (thanks!), and the idea that different 'genres' are associated with different colors is an interesting question, but not one I don't entirely get behind. I see why you asked it, since it can lead to some interesting comments; what strikes me is maybe **how** it is played/appreciated? For example, a 'symphony' or 'orchestra' seem modern to me, in that there is some sense of structure and theater/an audience; and I would maybe associate 'live jazz' or a 'live dj set' with teal, something that is happening live or is a process of 'discovery'. Just a thought here. 

Here's what I would say about music though, sort of as it relates to this discussion: some music I would categorize as 'emancipatory art', in the sense that it is music that has the power to transform ones understanding of the world or oneself merely by listening to it, in a spiritual sort of way. (This would also include other forms of 'art', but art that we absorb, like music or a painting, not art that we *do*) I would lump in here what you seem to be saying about Jacob Collier (maybe you don't agree), although this is the first time I've listened to him, and I find it somewhat emancipatory on the first listen, which also emphasizes that this to some degree depends on taste. (This is always a recurring question though: good art for whom? or emancipatory art for whom?) But I would say that some music definitely does have this quality, and that's what I find really interesting about music in my life. Maybe this is true of music in all time periods, perhaps this is even a key goal of music- to get the listener to experience some kind of 'beauty'. Here are some examples I would give from my end as music that is 'emancipatory'. (I would also say that this is in my experience really rare, somewhere around 5% that I listen to, and also that it continues to happen, it's not just listening to it once or twice)

Album wise: 
https://www.youtube.com/watch?v=-tT32VTll5M

Song wise:  
https://www.youtube.com/watch?v=hvoOhqybvlU
https://www.youtube.com/results?search_query=neil+frances+teardrops
https://www.youtube.com/watch?v=6VFoh5AbpBM
https://www.youtube.com/watch?v=hvoOhqybvlU 

Set wise: 
https://www.youtube.com/watch?v=5mpafLYHVd0

**Reply by asimong** [reply]
Appreciating your comments, @matthewmccarthy11 — hope to get back to them in the new year! Have a happy one...

**Reply by matthewmccarthy11** [reply]
Following up here as it relates to 'metamodern music' in two different ways: 

First : It's hard to imagine a person who doesn't listen to music; and it's interesting to consider how music is to some degree a reflection of ones own way of experiencing the world. As an 'interconnected world', all kinds of music are available and, in my case, if there is someone who has a similar taste in music in me, or knows the people I listen to, it seems as though there is something significant that we share, whatever that is. It points to a common experience, and in a way leads to a significantly greater trust in the person. So there is something interesting here where 'all the worlds music is available' and it also is a reflection, to some degree, of or own experiences of the world. In a way music is a great metaphor for our moment in a lot of ways: there is not 'one best genre' (or worldview) but a mix of them that people feel they align with, and they can all coexist. (The metaphor can also be taken in other ways but looking at the positive side here)

Adding in here, music is also to some degree a good metaphor for IFS; our liking different genres being something that speaks possibly to different parts of who we are

Second: Related to the point above here and what I mentioned before, 'metamodern music' being concerned not with music as something we passively 'listen to' but engage with on a deeper level, whether that includes playing it and exploring and feeling things like 'emergence' or dancing or mediating to it. Related, that 'metamodern' music (or music now) can increasingly have a 'spiritual element or form' and be a place to draw a significant amount of meaning. Maybe this is always the case with music, but it's something that I would highlight with some music that I hear (and seems to be the case with Jacob Collier), and was what I was getting at with calling it emancipatory, and I see as connected to the point above about many kinds of music.
